tests/data/serialization/ndjson/test_export_video_objects.py

```diff
@@ -585,6 +585,10 @@ def video_serialized_bbox_label():
     }
 
 
+#ignore uuid because we randomize if there was none
```

Comment:
For `test_export_video_objects.py` and test_image.py`, was there a reason why the optimizations were put in place? Now that we don't need to ignore "page" and "unit", I don't think we need to create a separate `IGNORE_KEYS` list just to hold one item, and it's creating unnecessary commit noise for these lines of code. I commend the efforts to make things more future-proof though!